PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -51,15 +50,16 @@ public async Task<IActionResult> CreateAsync([FromBody] Birthday input)
         {
             return BadRequest("Validation error.");
         }
-        await _dbcontext.Birthdays.AddAsync(input);
-       await _dbcontext.SaveChangesAsync();
-
-        return Ok(input);
+        var birthdayId = await _birthdayService.CreateAsync(input);
+        
+        
+        // ReSharper disable once Mvc.ActionNotResolved
+        return CreatedAtAction(nameof(GetByIdAsync), new { id = birthdayId }, input);
```

Comment:
Here you have to get Birthday using method `_birthdayService.GetByIdAsync(..)`;
And you have to return this Birthday using `return Ok(..)`.